### DISCUSSION TITLE
Can not access web page

### DISCUSSION THREAD

**Discussion by Ruben1308** [dingo35/SmartEVSE-3.5#124] [Q&A]
I connected my new SmartEVSE to the WIFI network but I'm not able to ping it or show the status page of the device (either by using the IP address or the device name). The IP address which is shown on the device looks valid and the device also shows the correct time. The device (name) is shown in the Network scanner of the ESET virus scanner on my laptop.

What could be the issue? 

Please note that I have only connected the device to the main power (for testing first)

Thanks for any ideas
Ruben Bussink

**Comment by dingo35** [comment]
What version are you on? It is shown in the bottom right corner of your display in the menu.

**Reply by Ruben1308** [reply]
The version is v3.6.9. This is the version which was installed on the device during delivery (last week).
Thank you for your reply.

Ruben

**Comment by dingo35** [comment]
There are no wifi bugs known on this version, but you could try setting wifi to disabled, exit menu, and enable wifi again.
Also try to address http:// instead of https://

**Reply by Ruben1308** [reply]
Hi Dingo, I will try your method once more. 
Is there a preferred browser?

Thanks
Ruben

**Comment by dingo35** [comment]
No they should all work!
